packages/tutorials/election-dapp/pact/election.pact

```diff
@@ -0,0 +1,161 @@
+(namespace 'free)
```

Comment:
This smart contract doesn't have any formal verification, do we want to introduce that in this tutorial or will that become too much?

Reply:
we should mention it and offer the user to implement it with an ***(optional-for-now)*** comment. Or later in the tutorial, we can come back to it and add it into a ***Best Practices section*** at the end of the tutorial.

Reply:
For the tutorial sake it might add value, however I would add that separately and align the tutorial's content. Currently it doesn't mention FV in the tutorial at all.